test/e2e/tests/test_bucket.py

```diff
@@ -85,6 +88,9 @@ def test_smoke(self, s3_client):
         exists = self.bucket_exists(s3_client, resource_name)
         assert exists
 
+        return (ref, resource_name, resource_data)
+
+    def _delete_smoke(self, s3_client, ref, resource_name):
```

Comment:
s/_delete_smoke/_delete_bucket/

(and I recommend moving it to module-level function)